docs/UserGuide.md

```diff
@@ -361,8 +378,18 @@ Edits the details of a specific task.
 
 Format: `editTask INDEX [en/ELDERLY_NAME]... [desc/DESCRIPTION] [date/DATE] [time/TIME] [recur/RECURRENCE_TYPE]`
 
+<div markdown="block" class="alert alert-info">
+
+:information_source: **Information**
+
+* If the date/recurrence type of a recurring task is edited, the updated date generated will be based on the edited date.
```

Comment:
I think this part could be a little clearer, "based on the the most recent edited date".